### DISCUSSION TITLE
Getting incompatible version

### DISCUSSION THREAD

**Discussion by alozic88** [mbround18/valheim-docker#304] [Help!]
Hi, 

with the recent game update, I'm getting an incompatible version when trying to log in. May I ask when will the "image: mbround18/valheim:latest" be updated to 0.148.6 and is the command "docker-compose up" enough to automatically update it?

Thanks!

edit: I just rebooted the server, ran "docker-compose up" and it worked. you can delete the thread, sorry for any inconvenience

**Comment by mbround18** [comment]
Expected,


They often update the client version before releasing a new version of the server. Suggestion is to pause updating your game on steam until the server version is pushed.

**Reply by kieron** [reply]
Is the server patch out yet? 🙂

Know it's nothing to do with the container of course! Just checking.

**Reply by mbround18** [reply]
![image](https://user-images.githubusercontent.com/12646562/112344257-271fd700-8c81-11eb-8923-2d49bac68587.png)
